ee/packages/pdf-worker/src/templates/ChatTranscript/components/MessageList.tsx

```diff
@@ -19,16 +20,21 @@ const styles = StyleSheet.create({
 	},
 });
 
+const messageLongerThanPage = (message: string) => message.length > 1200;
```

Comment:
How can we know that 1200 characters are longer than a page? 🤔

Reply:
Oh we actually don't 😬 

This was based on tests I performed where this was the "ideal" char limit based on the font + size + space available.

The max amount of chars a page with our settings allowed was around 1600. I'm here reducing it a little bit so it can "ideally" fit an image in the same page alongside big text.

Since all PDF sizing is standard for our case, as we dont allow customizations, this should be fine.